indico/modules/events/papers/operations.py

```diff
@@ -139,16 +158,17 @@ def create_paper_revision(paper, submitter, files):
                        _contribution=paper.contribution)
         pf.save(f.file)
     db.session.flush()
+    db.session.expire(revision._contribution, ['_paper_last_revision'])
```

Comment:
It's needed. Otherwise:
```
File "/home/iliastrichopoulos/dev/indico/indico/modules/events/papers/notifications.py", line 27, in notify_paper_revision_submission
    event = revision.paper.event_new
AttributeError: 'NoneType' object has no attribute 'event_new'
```